patches/minecraft/net/minecraft/data/recipes/RecipeCategory.java.patch

```diff
@@ -0,0 +1,50 @@
+--- a/net/minecraft/data/recipes/RecipeCategory.java
++++ b/net/minecraft/data/recipes/RecipeCategory.java
+@@ -1,23 +_,36 @@
+ package net.minecraft.data.recipes;
+ 
+-public enum RecipeCategory {
+-   BUILDING_BLOCKS("building_blocks"),
+-   DECORATIONS("decorations"),
+-   REDSTONE("redstone"),
+-   TRANSPORTATION("transportation"),
+-   TOOLS("tools"),
+-   COMBAT("combat"),
+-   FOOD("food"),
+-   BREWING("brewing"),
+-   MISC("misc");
++
++public enum RecipeCategory implements net.minecraftforge.common.IExtensibleEnum {
++   BUILDING_BLOCKS("building_blocks", net.minecraft.world.item.crafting.CraftingBookCategory.BUILDING),
++   DECORATIONS("decorations", net.minecraft.world.item.crafting.CraftingBookCategory.MISC),
++   REDSTONE("redstone", net.minecraft.world.item.crafting.CraftingBookCategory.REDSTONE),
++   TRANSPORTATION("transportation", net.minecraft.world.item.crafting.CraftingBookCategory.MISC),
++   TOOLS("tools", net.minecraft.world.item.crafting.CraftingBookCategory.EQUIPMENT),
++   COMBAT("combat", net.minecraft.world.item.crafting.CraftingBookCategory.EQUIPMENT),
++   FOOD("food", net.minecraft.world.item.crafting.CraftingBookCategory.MISC),
++   BREWING("brewing", net.minecraft.world.item.crafting.CraftingBookCategory.MISC),
++   MISC("misc", net.minecraft.world.item.crafting.CraftingBookCategory.MISC);
+ 
+    private final String f_244582_;
+ 
+-   private RecipeCategory(String p_251010_) {
++   private final net.minecraft.world.item.crafting.CraftingBookCategory craftingBookCategory;
```

Comment:
This makes no sense, a recipe category is a separate thing from a crafting book category. Why not have the cooking book category in here as well then? Remove.